gamemodes/terrortown/gamemode/server/sv_entity.lua

```diff
@@ -2,6 +2,12 @@
 -- @ref https://wiki.facepunch.com/gmod/Entity
 -- @class Entity
 
+-- Caps taken from here: https://github.com/ValveSoftware/source-sdk-2013/blob/55ed12f8d1eb6887d348be03aee5573d44177ffb/mp/src/game/shared/baseentity_shared.h#L21-L38
+FCAP_IMPULSE_USE = 16
+FCAP_CONTINUOUS_USE = 32
+FCAP_ONOFF_USE = 64
+FCAP_DIRECTIONAL_USE = 128
```

Comment:
Documenting these is a good idea, but we don't need them accessible via code IMO.

We can simply precompute these into a single value and use that value below instead of the bitwise-or

Reply:
> You can pass these to the function if you want to get a specific check, so we need them.

Ah fair.

> How would I document them? We never documented enums so far

I meant documenting what fcaps we use in general (e.g. in a full text comment) not as full doc comment. It's like it is now though